app/models/order.rb

```diff
@@ -11,4 +11,8 @@ def grandtotal
   def total_items
     item_orders.sum('quantity')
   end
+
+  def all_fulfilled?
+    item_orders.all? { |item_order| item_order.status == "fulfilled" }
```

Comment:
Great use of a oneline do block here!

`item_orders.all? { |item_order| item_order.status == "fulfilled" }` is the same as
```
item_orders.all? do |item_order|
   item_order.status == "fulfilled"
end
```
right?

Reply:
Yes! That's right! :)